subprojects/core/src/main/java/org/gradle/initialization/DefaultGradleLauncher.java

```diff
@@ -228,17 +234,28 @@ public void stop() {
     private class LoadBuild implements RunnableBuildOperation {
         @Override
         public void run(BuildOperationContext context) {
-            // Evaluate init scripts
-            initScriptHandler.executeScripts(gradle);
+            try {
+                // Evaluate init scripts
+                initScriptHandler.executeScripts(gradle);
+
+                // Build `buildSrc`, load settings.gradle, and construct composite (if appropriate)
+                settings = settingsLoader.findAndLoadSettings(gradle);
 
-            // Build `buildSrc`, load settings.gradle, and construct composite (if appropriate)
-            settings = settingsLoader.findAndLoadSettings(gradle);
+            } finally {
+                context.setResult(RESULT);
+            }
         }
 
         @Override
         public BuildOperationDescriptor.Builder description() {
-            return BuildOperationDescriptor.displayName(contextualize("Load build")).
-                parent(getGradle().getBuildOperation());
+            return BuildOperationDescriptor.displayName(contextualize("Load build"))
+                .details(new LoadBuildBuildOperationType.Details(){
+                    @Override
+                    public String getBuildPath() {
+                        return gradle.getIdentityPath().toString();
+                    }
+                })
+                .parent(getGradle().getBuildOperation());
```

Comment:
Is this necessary? I'm surprised that this isn't the default.

Reply:
removed. leftover from origin impl

Reply:
Looks to be still there.

Reply:
turns out it is necessary so I moved it back in. removing it changes the order of the build operations as we have them now. For now I keep them as is.

Reply:
I need much more explanation.

What is the parent if we omit this? What is the _problem_?

Reply:
I looked into it. We want to change it. I have raised a separate follow card internally to do this.